charts/shoot-core/components/charts/coredns/templates/coredns-deployment.yaml

```diff
@@ -104,7 +105,11 @@ spec:
             path: /ready
             port: 8181
             scheme: HTTP
-          initialDelaySeconds: 5
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          timeoutSeconds: 5
+          successThreshold: 1
+          failureThreshold: 3
```

Comment:
Is there a reason why you specify `periodSeconds`, `successThreshold` and `failureThreshold`? The values are the defaults as far as I can tell.

Reply:
Yes they are the default. The main reason is to have them here in the template because it is not always easy to find the default values in the documentation/code. And as I was touching other fields from the readiness probe, I have decided to explicitly set those, too.

Reply:
Ok, fine with me.